cmd/create/accountroles/cmd.go

```diff
@@ -79,13 +78,15 @@ func init() {
 		"",
 		"The arn path for the account roles",
 	)
+	flags.MarkHidden("role-path")
```

Comment:
Why are hiding these role/policy paths? They are still used for the account roles and policies, so we need them both here _and_ in the operator roles/policies

Reply:
the path feature is not yet implemented (not fully and has lots of bugs) and we do not want to release it yet. So after discussing with @pvasant we decided to hide it until the whole implementation is ready.

Reply:
Ah, that's a good idea. Can you do it in a separate commit please? It can be in the same PR, but in a separate commit in case one or the other needs to be reverted it doesn't drag the rest of the changes in.

Reply:
done